src/extensions/askUserForParam.ts

```diff
@@ -13,7 +13,7 @@ export function checkResponse(response: string, spec: Param): boolean {
   let valid = true;
   let responses: string[];
 
-  if (spec.required && !response) {
+  if (spec.required && (response == "" || response == undefined)) {
```

Comment:
This is a little bit ugly, but necessary to cover both the case where a param is completely omitted from the .env file, and the case where a param is set to nothing in the .env file.

Reply:
So, looking at this again, I think the real issue is that the typing here is pretty wonky. The YAML parser and .ENV parser we use cast strings that look like numbers or booleans to actual numbers or booleans, and we don't always cast those back to strings before passing them around. In this case, this was breaking if you ever tried to set a param to "false" or "0" - the parsers would cast those to  boolean false or number 0, and suddenly `!response == true`. 

If you think this is too hacky, I can try to fix these typing issues in this PR as well.

Reply:
It seems like we're trying to exclude the `response=false` from falling into this if block? Maybe we should explicitly specify that?

i.e. (!response && response !== false)

Reply:
YAML parsing is full of weird surprises..